themes/bootstrap3/templates/RecordTab/overdriveholdings.phtml

```diff
@@ -66,16 +67,20 @@
         <?php endif; ?>
       <?php endif; ?>
       <?php if ((!$checkedOut && !$hold) && ($supportsPatronActions || isset($urls[0]['url']))): ?>
-        <?php if (($avail->data->copiesAvailable ?? 0) == 0): ?>
+        <?php if ($copiesAvailable == 0): ?>
           <p>
-            <a class="btn btn-primary placehold" data-lightbox title="<?=$this->transEsc('request_place_texts')?>"
-              href="<?=$supportsPatronActions && count($urls) ?
+            <?php
+              $holdLink = $supportsPatronActions && count($urls) ?
                 "$hold_url?od_id=" . urlencode($od_id) . '&rec_id=' . urlencode($rec_id) . '&action=holdConfirm' :
-                $urls[0]['url']
-              ?>">
-              <?=$this->icon('place-hold', 'icon-link__icon')?>
-              <?=$this->transEsc('od_but_hold')?>
-            </a>
+                $urls[0]['url'] ?? null;
+            ?>
+            <?php if ($holdLink): ?>
+              <a class="btn btn-primary placehold" data-lightbox title="<?=$this->transEsc('request_place_texts')?>"
+                href="<?=$this->escapeHtmlAttr($holdLink)?>">
```

Comment:
I can't test this part as our institution does not `$supportsPatronActions`, but the code looks reasonable.